tck/src/main/java/org/eclipse/microprofile/lra/tck/LraTckConfigBean.java

```diff
@@ -34,6 +34,45 @@
      */
     private static final Long LRA_TIMEOUT_MILLIS = 50000L;
 
+    /**
+     * Name of the system property that is used as factor adjusting timeout values
+     * used in the testsuite.
+     */
+    private static final String LRA_TCK_TIMEOUT_FACTOR_PROPETY_NAME = "lra.tck.timeout.factor";
+
+    /**
+     * Name of the system property which is used to configure the TCK base url.
```

Comment:
This is actually a config property - https://github.com/eclipse/microprofile-lra/pull/313/files#diff-d7c8d58d8c6caa6639e33500b0985073R96. And we should make all TCK properties a config properties (previous is really a system property only).

Reply:
aghrr... I missed the point of this comment before. Now I got it. I changed the text for 'config property'.